internal/apiresource/triggerbinding.go

```diff
@@ -57,13 +56,7 @@ func (*TriggerBinding) createNewResource(irtriggerbinding tekton.TriggerBinding)
 	return triggerBinding
 }
 
-// ConvertToClusterSupportedKinds converts the object to supported types if possible.
-func (tb *TriggerBinding) ConvertToClusterSupportedKinds(obj runtime.Object, supportedKinds []string, otherobjs []runtime.Object, _ irtypes.EnhancedIR) ([]runtime.Object, bool) {
-	supKinds := tb.GetSupportedKinds()
-	for _, supKind := range supKinds {
-		if common.IsStringPresent(supportedKinds, supKind) {
-			return []runtime.Object{obj}, true
-		}
-	}
-	return nil, false
+// convertToClusterSupportedKinds converts the object to supported types if possible.
+func (tb *TriggerBinding) convertToClusterSupportedKinds(obj runtime.Object, supportedKinds []string, otherobjs []runtime.Object, _ irtypes.EnhancedIR) ([]runtime.Object, bool) {
+	return []runtime.Object{obj}, true
```

Comment:
same as above.

Reply:
this function is to moving between kind concepts if there is a similar concept supported in the cluster. In case there is no equivalent concept and the resource is required, it is better to retain the object. It is for the final writer to decide whether to serialize it or not.